deploy/managed-upgrade-operator-config/4.5/10-managed-upgrade-operator-configmap.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: managed-upgrade-operator-config
+  namespace: openshift-managed-upgrade-operator
+data:
+  config.yaml: |
+    configManager:
+      source: OCM
+      ocmBaseUrl: ${OCM_BASE_URL}
+      watchInterval: 60
+    maintenance:
+      controlPlaneTime: 90
+      ignoredAlerts:
+        controlPlaneCriticals:
+        # etcdMembersDown - OSD-6138
+        - etcdMembersDown
+        # ClusterOperatorDown - OSD-6330
+        - ClusterOperatorDown
+        # ClusterOperatorDegraded - OSD-6330
+        - ClusterOperatorDegraded
+        # CloudCredentialOperatorDown - BZ 1889540
+        - CloudCredentialOperatorDown
+    scale:
+      timeOut: 30
+    upgradeWindow:
+      delayTrigger: 30
+      timeOut: 120
+    nodeDrain:
+      timeOut: 45
+      expectedNodeDrainTime: 8
+    healthCheck:
+      ignoredCriticals:
```

Comment:
Do we need add comments for those alert?

Reply:
@wanghaoran1988 - This PR was about the silencing of alerts during controlplane upgrade. The alerts for healthcheck to be reviewed separately as part of [OSD-6691](https://issues.redhat.com/browse/OSD-6691). For now, we are maintaining the healthcheck alerts to be present as it is.